runtime/onert/api/include/nnfw.h

```diff
@@ -77,8 +77,13 @@ typedef enum {
   NNFW_TYPE_TENSOR_QUANT8_ASYMM = 2,
   /** A tensor of boolean */
   NNFW_TYPE_TENSOR_BOOL = 3,
+
+  /** A tensor of 64 bit signed integer */
+  NNFW_TYPE_TENSOR_INT64 = 4,
+
   /** A tensor of 8 bit unsigned integer */
-  NNFW_TYPE_TENSOR_UINT8 = 4,
+  NNFW_TYPE_TENSOR_UINT8 = 5,
```

Comment:
AFAIK, enum value of api shouldn't be changed.

Reply:
When I try to append to enumeration that you commented, I have the problem about range check in nnpacket_run.cc 
```
auto verifyInputTypes = [session]() {
      uint32_t sz;
      NNPR_ENSURE_STATUS(nnfw_input_size(session, &sz));
      for (uint32_t i = 0; i < sz; ++i)
      {
        nnfw_tensorinfo ti;
        NNPR_ENSURE_STATUS(nnfw_input_tensorinfo(session, i, &ti));
        if (ti.dtype < NNFW_TYPE_TENSOR_FLOAT32 || ti.dtype > NNFW_TYPE_TENSOR_UINT8)
        {
          std::cerr << "E: not supported input type" << std::endl;
          exit(-1);
        }
      }
    };

    auto verifyOutputTypes = [session]() {
      uint32_t sz;
      NNPR_ENSURE_STATUS(nnfw_output_size(session, &sz));

      for (uint32_t i = 0; i < sz; ++i)
      {
        nnfw_tensorinfo ti;
        NNPR_ENSURE_STATUS(nnfw_output_tensorinfo(session, i, &ti));
        if (ti.dtype < NNFW_TYPE_TENSOR_FLOAT32 || ti.dtype > NNFW_TYPE_TENSOR_UINT8)
        {
          std::cerr << "E: not supported output type" << std::endl;
          exit(-1);
        }
      }
    };
```
Please give your comment. @ragmani @wateret

Reply:
I think changing code from `nnpackage_run` makes more sense.

Reply:
I agree with @wateret 's opinion

Reply:
I got it.